### ISSUE TITLE
Size doesn't get set when the blob is deferred

### ISSUE THREAD

**Issue by Sid-Turner-Ellis** [RonRadtke/react-native-blob-util#241]
```
const blob = RNBlobUtil.polyfill.Blob.build(...)
const part = blob.slice(0, 100)
```
then there will be no size property on the part, it seems to because of [this line](https://github.com/RonRadtke/react-native-blob-util/blob/132051c1204051f5148a0b4bfede00ed99f47c44/polyfill/Blob.js#L134) where the return is done early for deferred blobs and the _invokeOnCreated method does not set the size.

Is this desired? just want to confirm before making a PR

**Comment by RonRadtke** [comment]
It is desired that the flag ist set.
Which return are you running in? This one: https://github.com/RonRadtke/react-native-blob-util/blob/132051c1204051f5148a0b4bfede00ed99f47c44/polyfill/Blob.js#L138 ?
Because then it would more relevant why defer = true.

_isReference  should not influence the behaviour of the size being set or not.

**Comment by Sid-Turner-Ellis** [comment]
I'm not sure i'm following - is it desired that there are times where the size property is not set on the blob?

**Comment by RonRadtke** [comment]
Yes there are cases.  If defer=True the size should not be returned
